### ISSUE TITLE
Image sizes

### ISSUE THREAD

**Issue by andybrewer** [magenta/magenta-js#438]
Is there documentation around using style transfer on larger images? The generated canvas image seems constrained to 300px wide.

The fox image is the content and the abstract art is the style, but I can't seem to get the rendered output to be in the same dimensions.

<img width="1045" alt="Screen Shot 2020-04-04 at 3 20 57 PM" src="https://user-images.githubusercontent.com/7093/78462521-d38b3680-7687-11ea-857a-f312f55c7c70.png">

**Comment by adarob** [comment]
@vdumoulin

**Comment by notwaldorf** [comment]
Based on the [code](https://github.com/magenta/magenta-js/blob/master/image/src/arbitrary_stylization/model.ts#L105) and the [demo](https://github.com/magenta/magenta-js/blob/master/image/demos/arbitrary_stylization.html#L34), i think the image needs to have a height of 256px. I agree that this should probably be documented a bit better.

(If you would like to send a PR just to update this size in the README, I'd love to merge it! 🥰)

**Comment by kfarr** [comment]
Yes it is not clear if there is any option to modify the output image dimensions or if this is "hard coded" into the model somehow. To be more generally useful for use in third-party applications control or at least documentation of output characteristics would be helpful.
